ironfish/src/telemetry/telemetry.ts

```diff
@@ -114,6 +114,26 @@ export class Telemetry {
   private metricsLoop(): void {
     Assert.isNotNull(this.metrics)
 
+    const peers = [...this.metrics.p2p_OutboundTrafficByPeer.entries()]
+    peers.forEach(([peer, meter]) => {
```

Comment:
Any reason not to just loop over the map itself using the iterator? This makes a copy, and then generates a function versus just using the iterator + for loop

Reply:
Yep, I think this was an artifact of playing around with some other map types. I'll change it to just use the iterator